src/Voron/Data/BTrees/Tree.cs

```diff
@@ -899,16 +896,11 @@ private bool TryUseRecentTransactionPage(Slice key, out TreePage page, out TreeN
             if (foundPage == null)
                 return false;
 
-            if (foundPage.Page != null)
-            {
-                // we can't share the same instance, Page instance may be modified by
-                // concurrently run iterators
-                page = new TreePage(foundPage.Page.Base, foundPage.Page.PageSize);
-            }
-            else
-            {
-                page = GetReadOnlyTreePage(foundPage.Number);
-            }
+            // we can't share the same instance, Page instance may be modified by
+            // concurrently run iterators
+            page = foundPage.Page != null ? 
+                new TreePage(foundPage.Page.Base, foundPage.Page.PageSize) : 
+                GetReadOnlyTreePage(foundPage.Number);
```

Comment:
What is the reason for making such changes? We had a comment put in relevant scope while now it's not so clear.

Reply:
Purely stylistic choice, reverted.